### ISSUE TITLE
This is really useful, but could it be updated to work with EF Core 3.0?

### ISSUE THREAD

**Issue by Going-Gone** [tkhadimullin/ef-core-custom-functions#1]

**Comment by Going-Gone** [comment]
```
protected override Expression Accept(ExpressionVisitor visitor)
        {
            if (!(visitor is IQuerySqlGenerator))
            {
                return base.Accept(visitor);
            }
            visitor.Visit(new SqlFragmentExpression("CONVERT(VARCHAR(MAX), DECRYPTBYKEY("));
            visitor.Visit(_value);
            visitor.Visit(new SqlFragmentExpression("))"));
            return this;
        }
```

**Comment by Going-Gone** [comment]
SqlFragmentExpression
IQuerySqlGenerator

Both seem to be not included in 3.0

**Comment by tkhadimullin** [comment]
so it seems, it can indeed be updated to work with ef 3.1. sorry to have taken my time. 
check out the `/feature/ef-3.1-version` branch
